catalog/spatial/csw/spatial-csw-transformer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -150,8 +150,28 @@
                             ref="inputTransformers"/>
     </reference-list>
 
+    <!-- QueryFilterTransformerProvider -->
+    <bean id="cswQueryFilterTransformerProvider"
+      class="org.codice.ddf.spatial.ogc.csw.catalog.transformer.CswQueryFilterTransformerProvider"/>
+
+    <reference-list id="queryFilterTransformerList"
+      interface="ddf.catalog.transform.QueryFilterTransformer"
+      member-type="service-reference"
+      availability="optional">
+        <reference-listener bind-method="bind"
+          unbind-method="unbind"
+          ref="cswQueryFilterTransformerProvider"/>
+    </reference-list>
+
+    <service ref="cswQueryFilterTransformerProvider" id="cswQueryFilterTransformerProviderService"
```

Comment:
Why does this have a ref and an id?

Reply:
`ref` is referencing to the bean that providing this service, while `id` is the id of the service, which is kinda optional.  http://aries.apache.org/modules/blueprint.html

Reply:
Gotcha, I've never put an id on a service.